Library/src/main/java/info/metadude/android/typedpreferences/BasePreference.java

```diff
@@ -0,0 +1,33 @@
+package info.metadude.android.typedpreferences;
+
+import android.content.SharedPreferences;
+
+/**
+ * A wrapper class for a boolean preference.
+ */
+class BasePreference {
```

Comment:
To keep the same style as the other classes please add `public`. The class also might be `abstract`, do you agree?